source/install/deprecated-features.rst

```diff
@@ -4,24 +4,26 @@ Removed and deprecated features for Mattermost
 This page describes features that are removed from support for Mattermost, or will be removed in a future update (deprecated), and provides early notice about future changes that might affect your use of Mattermost. This information is subject to change with future releases, and might not include each deprecated feature.
 
 Removed features in upcoming versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------------------
 
 Mattermost Server v8.0.0
-^^^^^^^^^^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 - Removing ``ExperimentalSettings.PatchPluginsReactDOM``. If this setting was previously enabled, confirm that:
-    1. All Mattermost-supported plugins are updated to the latest versions.
-    2. Any other plugins have been updated to support React 17. See the `Important Upgrade Notes <https://docs.mattermost.com/upgrade/important-upgrade-notes.html>`__ for v7.7 for more information.
+
+  - All Mattermost-supported plugins are updated to the latest versions.
+  - Any other plugins have been updated to support React 17. See the `Important Upgrade Notes <https://docs.mattermost.com/upgrade/important-upgrade-notes.html>`__ for v7.7 for more information.
+- Disabling Insights for all new instances and for existing servers that upgrade to Mattermost v8.0. You can re-enable Insights with an environment variable via ``MM_FEATUREFLAGS_INSIGHTSENABLED=true``.
```

Comment:
John mentioned that we don't want to mention the environment variable flag in the docs.

Reply:
Thanks, @amyblais! I'll pull that detail out of this PR.